tests/repositories/fixtures/pypi.org/json/clikit.json

```diff
@@ -1,6 +1,6 @@
 {
   "info": {
-    "author": "Sébastien Eustace",
+    "author": "S\u00e9bastien Eustace",
```

Comment:
Is this one expected?

Reply:
This is expected given https://simplejson.readthedocs.io/en/latest/#simplejson.dumps. There is no provision to disable this upstream -- I could add a argument if it's desired.

On the other hand, the spec allows for either encoded or non-encoded UTF-8 characters. This file is going to decode the same. If it's a bit too far (vs just re-indenting), I can roll it back until we get the ability to disable these escapes.

Reply:
It turns out I was looking at an old version of the hook and upstream already has support for turning this off.

https://github.com/pre-commit/pre-commit-hooks/blob/v4.0.1/pre_commit_hooks/pretty_format_json.py#L77